### ISSUE TITLE
[Suggestion] Trusted admins in config?

### ISSUE THREAD

**Issue by LaFesta1749** [KarlOfDuty/SCPDiscord#153]
I suggest (if possible) to add Trusted Admins SteamIDs so that their commands are not logged in the staff channel.
Only the added SteamIDs would not be logged while the others that are not added will be logged.

Is this possible?

**Comment by KarlOfDuty** [comment]
I would presonally not recommend that you do this.

It can be a security issue if only some users are logged, one couldnt be sure if for instance someone is cheating or if one of the unlogged users gave them a speed effect.

It can make it very annoying when you want to go back in the log later and check what happened at a specific time.

If this is still something you want to do you can use the cancel message regex which was added in the previous patch. I believe there is an example in the overrides.yml file in the language directory.

Basically copy the messages you want to cancel in there and then add the steam ids of the users into the cancel_regex parameter. If the message does not have a steamid in it you may have to add it to the message first though.

**Comment by KarlOfDuty** [comment]
Did this answer your question? If there is nothing else, you can close the issue.
